codelists/actions.py

```diff
@@ -24,6 +24,7 @@ def create_old_style_codelist(
     description,
     methodology,
     csv_data,
+    coding_system_version_slug=None,
```

Comment:
I made them all optional so that things could continue to work before we got as far as importing new coding system versions (with the idea that we'd be able to merge these PRs incrementally).  I think you're right though.